src/components/tasks/crud/createTask.js

```diff
@@ -57,60 +58,69 @@ const basicSwal = {
 
 const MySwal = withReactContent(Swal)
 
-export const createTask = (tasks, reload, setReload) => {
+export const createTask = (onCreateTask, setReload) => {
     MySwal.fire({
         ...basicSwal,
         html: <AddDescription
             handleChange={{'md': handleMarkdownChange, 'text': handleTextChange, 'file': handleFileChangeDescription}}/>
     }).then(result => {
         if (result.isConfirmed) {
-            createTests(tasks, reload, setReload)
+            createTests(onCreateTask, setReload)
         }
     })
 }
 
 
 const handleManualTestsChange = (testCases) => {
     delete task['testsBase64']
+    task['testsBase64'] = btoa(JSON.stringify(testCases))
```

Comment:
Czy btoa to nasza funkcja czy jakaś z biblioteki?

Reply:
Jsowa funkcja do robienia base64

Reply:
Czy nie lepiej już użyć tych z `Base64.js`? Wtedy jak się okaże, że się walnęliśmy, to walniemy się wszędzie tak samo